daemon/api.go

```diff
@@ -132,17 +132,19 @@ var (
 	}
 
 	snapsCmd = &Command{
-		Path:   "/v2/snaps",
-		UserOK: true,
-		GET:    getSnapsInfo,
-		POST:   postSnaps,
+		Path:     "/v2/snaps",
+		UserOK:   true,
+		PolkitOK: "io.snapcraft.snapd.manage-snaps",
```

Comment:
How fine grained do we plan this to be?  Gut feeling is that "manage-snaps" might pretty much cover the entirety of snapd's functionality, and as such simply "manage" might be enough. Alternatively, do we want to distinguish installations/removals from configuration?  If so, we probably need more specific names.

So, either something more specific, or if we plan for it to be widely scoped (which sounds fine in practice), perhaps simply "manage" might be enough.

Perhaps going wide is the sanest route for now, actually, as many of the actions are in fact different parameters on the request, so we'll need to change this model a bit if we want to be more fine grained.

Reply:
If you want, I can change the name of the polkit action to something more specific (any particular preference?), and we can look at splitting the other operations off into other action IDs if needed.

I guess the main question is whether admins would want to grant some of these permissions to users but not others.  For the policy defaults, I don't think it matters much though.